UM/PackageManager.py

```diff
@@ -291,6 +296,24 @@ def getAllInstalledPackageIDs(self) -> Set[str]:
 
         return all_installed_ids
 
+    ## Get a list of tuples that contain the package ID and version.
+    #  Used by the Marketplace to check which packages have updates available.
+    def getAllInstalledPackageIdsAndVersions(self) -> List[Tuple[str, str]]:
+        package_ids_and_versions = []  # type: List[Tuple[str, str]]
+        all_installed_ids = self.getAllInstalledPackageIDs()
+        for package_id in all_installed_ids:
+            if package_id in self._bundled_package_dict:
+                continue
+            if package_id in self._application.getRequiredPlugins():
```

Comment:
A required plugin is bundled by definition. So this check will never do anything

Reply:
I'll remove this check then.